std_msgs/Header header
int64 altitude_check_depth   # altitude_check_depth reading
